tests/Mutator/Unwrap/UnwrapArrayFilterTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Tests\Mutator\Unwrap;
+
+use Infection\Tests\Mutator\AbstractMutatorTestCase;
+
+/**
+ * @internal
+ */
+final class UnwrapArrayFilterTest extends AbstractMutatorTestCase
+{
+    /**
+     * @dataProvider provideMutationCases
+     */
+    public function test_mutator($input, $expected = null): void
+    {
+        $this->doTest($input, $expected);
+    }
+
+    public function provideMutationCases(): \Generator
+    {
+        yield [
+            <<<'PHP'
+<?php
+
+$a = array_filter(['A', 1, 'C'], 'is_int');
+PHP
+            ,
+            <<<'PHP'
+<?php
+
+$a = ['A', 1, 'C'];
+PHP
+        ];
+
+        yield [
+            <<<'PHP'
+<?php
+
+$a = array_filter(['A', 1, 'C'], function($var): bool {
```

Comment:
could you please add more tests (like parameter as a variable, function with lower/upper cased characters and so on.)

See the list of cases in https://github.com/infection/infection/blob/master/tests/Mutator/Regex/PregQuoteTest.php

same for `array_map`

Reply:
Nice catch, wrongly capitalized was the problem :)

Reply:
There's `$node->name->toLowerString()` of use.